### ISSUE TITLE
Rename datatype UuidArrayOfHLAbyte16 to UUID

### ISSUE THREAD

**Issue by lofstrandbjorn** [AMSP-04/NETN-BASE#5]
There is extensive use of the UUID concept in all NETN FOM modules. A simplified name such as UUID and perhaps also for arrays of UUID to ArrayOfUUID would improve readability.

**Comment by bergtwvd** [comment]
Agree. Also code generators can leverage a standard naming. E.g. for Java generate the class UUID rather than byte[].

Also add information about the encoding of the UUID (not the HLA encoding, but the RFC4122 encoding).

**Comment by lofstrandbjorn** [comment]
Suggested sematics for datatype description:

"UUIDs are exchanged as a byte array and are encoded according to RFC 4122, section 4.1.2 using 16 bytes. Also referred to as Variant 1 or RFC 4122/DCE 1.1 UUIDs.

For example, 00112233-4455-6677-8899-aabbccddeeff is encoded as the bytes 00 11 22 33 44 55 66 77 88 99 aa bb cc dd ee ff."

**Comment by lofstrandbjorn** [comment]
Do we want to do this for Ed B???

**Comment by lofstrandbjorn** [comment]
I added the UUID datatype but will keep the UuidArrayOfHLAbyte16.

**Comment by OscBer97** [comment]
All occurrences of UuidArrayOfHLAbyte16 are renamed to UUID now, see https://github.com/AMSP-04/NETN-BASE/issues/18#issuecomment-1249104801
